_posts/2019-03-13-activeadmin-vs-administrate.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "ActiveAdmin vs Administrate"
+date: 2019-03-13 09:00:00
+categories: ["security" "gems"]
+author: "rdormer"
+---
+
+One of the first complications that most webapps of any complexity will run into is the need for privileged users who can do things that normal users can't or shouldn't.  Before too long, you're headed towards writing your very own administrative interface.  This is not only extra work to do, but can be tricky to do without compromising the security of the application you're administering.  Most Rails developers will be familiar with this story, and Rails being Rails, it turns out that there are a couple of good options for extending your existing applications with a pre-generated, customizable admin console.
+
+Specifically, you have two good choices in the form of ActiveAdmin and Administrate.  Both are implemented as engines, distributed as Gems, and make it dead easy to quickly develop an admin console.
```

Comment:
"... ActiveAdmin and Administrate"
@rdormer It would be nice to links their repos there

Reply:
@rdormer Maybe you can mention that there're more options like Rails Admin (https://github.com/sferik/rails_admin) but you find these two options to be the best. 
I think we used rails admin in the past but we prefer administrate